code/muilty_rtspsender_gui/rtsp_server.py

```
@@ -0,0 +1,1830 @@
+            # MediaMTX 방식: FFmpeg → RTMP → MediaMTX → RTSP
+            rtmp_port = 1935 + stream_id  # 각 스트림별 RTMP 포트
+            rtsp_port = port              # 각 스트림별 RTSP 포트
+            
+            # MediaMTX 연결 상태 사전 확인
+            mediamtx_ready = False
+            try:
+                with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+                    s.settimeout(2)
+                    result = s.connect_ex(('127.0.0.1', rtmp_port))
+                    mediamtx_ready = (result == 0)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)